app/policies/theblog/page_policy.rb

```diff
@@ -0,0 +1,11 @@
+module Theblog
+  class PagePolicy < ApplicationPolicy
```

Comment:
both classes are completely similar. I would inject one more class(ContentPolicy) into the hierarchy and then inherit from it